tasks/main.yaml

```diff
@@ -4,6 +4,7 @@
   group:
     name: '{{ kafka_group }}'
     state: present
+  when: kafka_create_user_group|bool == true
```

Comment:
Linting issue: "Don't compare to literal True/False".

Similar change required in other lines.

Reply:
That's been fixed.